src/Breadcrumb.php

```diff
@@ -59,9 +59,9 @@ final class Breadcrumb implements \JsonSerializable
     public const LEVEL_ERROR = 'error';
 
     /**
-     * This constant defines the critical level for a breadcrumb.
+     * This constant defines the fatal level for a breadcrumb.
      */
-    public const LEVEL_CRITICAL = 'critical';
```

Comment:
Thank you for the contribution. However, this is a breaking change as the constant is `public` and may be used by someone. Please add a new constant that defines the correct value and mark as deprecated this one

Reply:
Thanks, will do that.

Reply:
Should the deprecated constant be still in the allowed array?

Reply:
Yes, if we don't allow the value anymore we would possibly still break existing applications

Reply:
Done.